### PR TITLE
Ticket 2: Get all topics endpoint

### PR DESCRIPTION
**Pull request by omobayed**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/topics.controller");
+const {handlePsqlErrors ,handleCustomErrors}= require("./errors");
+
+const app = express();
+
+app.use(express.json());
```
Comment (Keb84) [inline app.js:7]: At this point express.json isn't required. Please remove this until you get to the point you need it 🙂

File: controllers/topics.controller.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const { selectAllTopics } = require("../models/topics.model")
+
+exports.getAllTopics = (req, res) => {
+    selectAllTopics().then((topics) => {
+        res.status(200).send({topics})
+    })
```
Comment (Keb84) [inline controllers/topics.controller.js:6]: Do you think a catch block would be useful here 🤔

**Review by Keb84 — APPROVED** [review]
Nice work for ticket 2 🙂 please have a look at the comments i've made and merge when ready 🚀

File: app.js (hunk 1 above)
Comment (omobayed) [reply app.js:7]: ok I will remove it in the next pr

File: controllers/topics.controller.js (hunk 2 above)
Comment (omobayed) [reply controllers/topics.controller.js:6]: yes, it is 
I will add it

**Pull request merged**